test/cli/lookup_test.py

```diff
@@ -341,44 +341,40 @@ def test_platform_lookup_builtin(tmpdir):
     ]
 
 
-@pytest.mark.skip  # TODO: performs additional lookups when run via symlink in CI
```

Comment:
It was not doing additional lookups. The tests incorrectly tested with the executable path but the CWD has preference.